app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,22 @@
+class CommentsController < ApplicationController
+  def create
+    user = current_user
+    comment = Comment.new(post_params)
```

Comment:
[OPTIONAL] Then in your `create` method, you can use the following code to create a new comment.

```
    def create
        @comment = @current_user.comments.new(comment_params)
        @comment.post_id = params[:post_id]

        ...
    end
```